cmd/draft/up.go

```diff
@@ -182,3 +186,32 @@ func (u *upCmd) run(environment string) (err error) {
 
 	return nil
 }
+
+func runTasks(kind, podName string) error {
+	taskList, err := tasks.Load(tasksYAMLFile)
+	if err != nil && err != tasks.ErrNoTaskFile {
```

Comment:
Would we want this to be an OR instead? I think the logic we want is if the error is non-nil OR the task file does not exist.

Also, if the task file does not exist, wouldn't we want to return an error?

Reply:
oh wait never mind... Let me think through this myself.

err == nil: (false && true) = false
err == SomeOtherErr: (true && true) = true
err == tasks.ErrNoTaskFile: (true && false) = false

You're right. Branch logic is hard sometimes.